Sources/SPTDataLoaderResponse.m

```diff
@@ -52,7 +51,6 @@ - (instancetype)initWithRequest:(SPTDataLoaderRequest *)request response:(nullab
     self = [super init];
     if (self) {
         _request = request;
-        _response = response;
```

Comment:
This was only being retained for the `description` method. Everything else was copied out of it.